### ISSUE TITLE
Django 3 support

### ISSUE THREAD

**Issue by ricco386** [incuna/django-wkhtmltopdf#168]
I have tried to run django-wkhtmltopdf with the latest Django development version and there is a change where `six` has been removed:  (django/django@4138481 / https://code.djangoproject.com/ticket/27753)

Version of Django 3.0 ~~due to be released until~~ was [released](https://docs.djangoproject.com/en/3.0/releases/3.0/) at 2. December.

```
ImportError: cannot import name 'six' from 'django.utils' (virtualenvs/lib/python3.7/site-packages/django/utils/__init__.py)

virtualenvs/lib/python3.7/site-packages/wkhtmltopdf/utils.py:24: ImportError
```

**Comment by yatabani** [comment]
+1

**Comment by aitorres** [comment]
+1

**Comment by eirrick** [comment]
+1

**Comment by xmontana** [comment]
+1

**Comment by maxpeterson** [comment]
Fixed in #170 / #173
